src/jemalloc.c

```diff
@@ -56,7 +56,8 @@ static malloc_mutex_t	arenas_lock;
  * arenas.  arenas[narenas_auto..narenas_total) are only used if the application
  * takes some action to create them and allocate from them.
  */
-arena_t			**arenas;
+JEMALLOC_ALIGNED(CACHELINE)
+arena_t			*arenas[MALLOCX_ARENA_MAX];
```

Comment:
Nice simplification.  It looks like there is an off-by-one issue though (should be ```MALLOCX_ARENA_MAX+1``` elements).  Incidentally, this mistake is due to ```MALLOCX_ARENA_MAX``` being more confusing than ```MALLOCX_ARENA_LIMIT```. :wink:  Will fix: #673